content/en/docs/tasks/security/webhook/index.md

```diff
@@ -0,0 +1,237 @@
+      name: istio-sidecar-injector
+      labels:
+        app: sidecarInjectorWebhook
+        chart: sidecarInjectorWebhook
+        heritage: Tiller
+        release: istio
+    webhooks:
+      - name: sidecar-injector.istio.io
+        clientConfig:
+          service:
+            name: istio-sidecar-injector
+            namespace: istio-system
+            path: "/inject"
+          caBundle: ""
+        rules:
+          - operations: [ "CREATE" ]
+            apiGroups: [""]
+            apiVersions: ["v1"]
+            resources: ["pods"]
+        failurePolicy: Fail
+        namespaceSelector:
+          matchLabels:
+            istio-injection: enabled
+    {{< /text >}}
+
+<!-- TODO (lei-tang): improve the UX for obtain ValidatingWebhookConfiguration -->
+1.  Open the `istio-webhook-config.yaml` configuration file, search `'kind: ValidatingWebhookConfiguration'` and save
+the `ValidatingWebhookConfiguration` of Galley to `galley-webhook.yaml`. The following
+is a `ValidatingWebhookConfiguration` in an example `istio-webhook-config.yaml` (only
+a part of the configuration is shown to save space).
+
+    {{< text yaml >}}
+    apiVersion: admissionregistration.k8s.io/v1beta1
+    kind: ValidatingWebhookConfiguration
+    metadata:
+      name: istio-galley
+      labels:
+        app: galley
+        chart: galley
+        heritage: Tiller
+        release: istio
+        istio: galley
+    webhooks:
+      - name: pilot.validation.istio.io
+        clientConfig:
+          service:
+            name: istio-galley
+            namespace: istio-system
+            path: "/admitpilot"
+          caBundle: ""
+        rules:
+          - operations:
+            - CREATE
+            - UPDATE
+            apiGroups:
+            - config.istio.io
+            ... SKIPPED
+        failurePolicy: Fail
+        sideEffects: None
+    {{< /text >}}
+
+1.  Use `istioctl` to enable the webhook configurations:
+
+    {{< text bash >}}
+    $ istioctl experimental post-install webhook enable --webhook-secret dns.istio-galley-service-account \
+        --namespace istio-system --validation-path galley-webhook.yaml \
+        --injection-path sidecar-injector-webhook.yaml
+    {{< /text >}}
+
+1.  To check that the sidecar injector webhook is working, verify that the webhook injects a
+sidecar container into an example pod with the following commands:
+
+    {{< text bash >}}
+    $ kubectl create namespace test-injection; kubectl label namespaces test-injection istio-injection=enabled
+    $ kubectl run --generator=run-pod/v1 --image=nginx nginx-app --port=80 -n test-injection
+    $ kubectl get pod -n test-injection
+    {{< /text >}}
+
+    The output from the `get pod` command should show the following output. The `2/2` value means that
+    the webhook injected a sidecar into the example pod:
+
+    {{< text plain >}}
+    NAME    READY   STATUS    RESTARTS   AGE
+    nginx-app   2/2     Running   0          10s
+    {{< /text >}}
```

Comment:
aren't this working without the steps here (e.g. using the default mode)?  how does user know if it is not using the default.

Reply:
@linsun I don't understand you question here?

Reply:
Done: added a step to verify that there are no existing webhook configurations for Galley and the sidecar injector before running the webhook enable command.